examples/basic-typescript/pages/[user]/[repo].tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import Link from 'next/link'
+import fetch from '../../libs/fetch'
```

Comment:
Is this a pattern worth continuing? Would it make more sense to do this here instead?
```suggestion
import fetch from 'isomorphic-unfetch';
```

Reply:
I have used `example/basic` as a template. Both examples are using a custom implementation of fetch that is directly returning `json`. 

https://github.com/zeit/swr/blob/master/examples/basic/pages/%5Buser%5D/%5Brepo%5D.js

Reply:
Doh, I forgot about the Json step!